packages/devextreme/js/__internal/ui/context_menu/m_context_menu.ts

```diff
@@ -968,13 +985,16 @@ class ContextMenu extends MenuBase {
       case 'visible':
         this._renderVisibility(args.value);
         break;
+      case 'disabled':
       case 'showEvent':
       case 'position':
       case 'submenuDirection':
         this._invalidate();
         break;
       case 'target':
-        args.previousValue && this._detachShowContextMenuEvents(args.previousValue);
+        if (args.previousValue) {
```

Comment:
const { name, value, previousValue } = args;

Also kindly add typing to _optionChanged() method